### PR TITLE
test: try to write something when init()

### PR DESCRIPTION
**Pull request by lance6716**
### What is changed and how it works?

record when we called `init()`. If time between `init()` and `main()` is long we know that some package has a slow `init()`.

### Check List 

Tests 

 - No code

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

**Comment by lance6716** [comment]
/run-unit-tests

seems TiDB with TLS failed

File: pkg/printinit/noimport.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package printinit
+
+// we expect this helper function to be called at very early stage of initialization, because this package has no
+// imports.
+// There are some failing CI indicating it takes at least 10 seconds to start a DM-master, we need to know if the OS
+// spawn processes too slowly or some init() are slow to finish.
+func init() {
+	println("init now")
```
Comment (lichunzhu) [inline pkg/printinit/noimport.go:8]: print a time.Now here？

**Review by lichunzhu — APPROVED** [review]
Rest LGTM

File: pkg/printinit/noimport.go (hunk 1 above)
Comment (lance6716) [reply pkg/printinit/noimport.go:8]: that will introduce an import, which will lower the probability that this init is at early stage of initialization

**Review by glorv — APPROVED** [review]
I'm suspicious of the effect of PR. But, since there are no other side effect, we can first merge it and see if it can help investigate the unstable issue.

**Comment by glorv** [comment]
I thinks we needn't merge this PR into release-2.0 branch

**Pull request merged**